e2e-tests/ukef/cypress/e2e/journeys/gef/submit-AIN-deal-with-issued-facilities-deal-cancelled-maker-unable-to-view-update-issue-facility-section/submit-AIN-deal-with-issued-facilities-deal-cancelled-maker-unable-to-view-update-issue-facility-section.spec.js

```diff
@@ -0,0 +1,111 @@
+import relative from '../../../relativeURL';
+import gefPages from '../../../../../../gef/cypress/e2e/pages';
+import { submitButton } from '../../../../../../gef/cypress/e2e/partials';
+import tfmPages from '../../../../../../tfm/cypress/e2e/pages';
+import tfmPartials from '../../../../../../tfm/cypress/e2e/partials';
+import MOCK_USERS from '../../../../../../e2e-fixtures/portal-users.fixture';
+import { MOCK_APPLICATION_AIN_DRAFT } from '../../../../../../e2e-fixtures/gef/mocks/mock-deals';
+import { anIssuedCashFacility } from '../../../../../../e2e-fixtures/mock-gef-facilities';
+import { PIM_USER_1, TFM_URL } from '../../../../../../e2e-fixtures';
+
+const { BANK1_MAKER1, BANK1_CHECKER1 } = MOCK_USERS;
+
+context('Portal to TFM deal submission', () => {
+  let dealId;
+  let deal;
+  const dealFacilities = [];
+
+  before(() => {
+    // inserts a gef deal
+    cy.insertOneGefDeal(MOCK_APPLICATION_AIN_DRAFT, BANK1_MAKER1).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+      deal = insertedDeal;
+      // updates a gef deal so has relevant fields
+      cy.updateGefDeal(dealId, MOCK_APPLICATION_AIN_DRAFT, BANK1_MAKER1);
+
+      cy.createGefFacilities(dealId, [anIssuedCashFacility()], BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities.push(createdFacilities.details);
```

Comment:
I've been running this locally & think you'll need this to run it on the pipeline
```suggestion
      cy.createGefFacilities(dealId, [anIssuedCashFacility({ facilityEndDateEnabled: true })], BANK1_MAKER1).then((createdFacilities) => {
```

Deal cancellation is only enabled in the `ff` e2e tests, in these tests we also have the facility end date feature enabled

Reply:
I am not quite sure what do you mean here. I tried to run test with { facilityEndDateEnabled: true }, but the test failed. Could anyone else help me with this as Mark is off today?